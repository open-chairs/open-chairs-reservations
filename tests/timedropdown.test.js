import React from 'react';
import ReactDOM from 'react-dom';
import TimeDropdown from '../client/components/TimeDropdown.jsx';
import { mount, shallow, render, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';


configure({ adapter: new Adapter() });

it('renders on page without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<TimeDropdown />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('should render correctly with no props', () => {
  const component = shallow(<TimeDropdown />);
  expect(component).toMatchSnapshot();
});
