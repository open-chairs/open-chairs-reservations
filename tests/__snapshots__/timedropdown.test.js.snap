// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should pass a value to onChange handler 1`] = `ShallowWrapper {}`;

exports[`should render correctly with no props 1`] = `ShallowWrapper {}`;
